### DISCUSSION TITLE
Add a way to xmog inaccessible items to players

### DISCUSSION THREAD

**Discussion by meudeusdosceus** [chromiecraft/chromiecraft#4551] [Suggestions]
Hello. 
For exemple i would like to mog my 2h mace in to a cool tauren big totem: 

![image](https://user-images.githubusercontent.com/102554765/207170801-0015bbec-4698-49e5-8eec-f29fe94f899b.png)

Probably the best way to make this happen its selling in CC store.

**Comment by Gultask** [comment]
Hey, @meudeusdosceus, there has definitely been some time passed since but check out the https://www.chromiecraft.com/product-category/the-moon-market/ 😁
